doc/commands/messageclass.md

```
@@ -0,0 +1,132 @@
+Download the message class content.
+
+```bash
+sapcli messageclass read ZMC_HELLO_WORLD [--output JSON|HUMAN]
+```
+
+* _--output HUMAN_ prints the message class in a human-readable table format (default)
+* _--output JSON_ prints the message class in JSON format
+
+Example HUMAN output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)